src/admin/js/user/SignIn.js

```diff
@@ -27,15 +27,14 @@ const signInSchema = buildSchema({
     },
 });
 
-const renderInput = (floatingLabelText, type, hintText) =>
-    ({ meta: { touched, error } = {}, input: { ...inputProps }, ...props }) => // eslint-disable-line react/prop-types
-        <TextField
-            errorText={touched && error}
-            {...{ floatingLabelText, type, hintText }}
-            {...inputProps}
-            {...props}
-            fullWidth
-        />;
+const renderInput = (field) => {
+    const { meta: { touched, error } = {}, input: { ...inputProps }, ...props } = field;
+    return(<TextField
```

Comment:
Btw, `input: { ...inputProps }` should be `input: inputProps` if you just want to rename it